README.md

```diff
@@ -22,17 +22,18 @@
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-# 📖 [Porfolio] <a name="about-project"></a>
+# 📖 [Emmacroy Technologies] <a name="about-project"></a>
 
-**[your_project__name]** is a...
+**[Emmacroy Technologies]** is my capstone project at <a href='https://www.microverse.org/'>Microverse</a> module one. It's an education website for web development. It currently contains only the Home and About pages, other pages like the contact page and registration pages will be create later. It is currently onsite but I intend to make it online too.
+I have used  the <a href='https://www.microverse.org/'>Creative Commons</a> design template by <a href='https://www.behance.net/adagio07'>Cindy Shin in Behance</a> for this project
```

Comment:
- [ ] I would kindly suggest that you move the credit you are giving to `Cyndy Shin` to the acknowledge section because that is what it is for.👌